JamesMudidi/iReporterApi
# Updating and refactoring code to pep8

#### What does this PR do?
* Refactor code to pep8 statndards
* Give appropriate error messages

#### Description of Task to be completed?
* changed the validation error message from "the parameter cannot be null" to "the parameter cannot be empty"
* changed the error names from "Index out of range" to "The ID provided is not in the system" so that a normal user can be catered for aswell
* Refactored all the code to pep8 standards

#### How should this be manually tested?
* On this Pull request navigate to the files changed tab to see the various files changed

#### Screenshots
![image](https://user-images.githubusercontent.com/39102955/51323427-3ab7cd80-1a79-11e9-9382-b1c8ba975188.png)

![image](https://user-images.githubusercontent.com/39102955/51323483-65098b00-1a79-11e9-9750-4cf5a4ec74d9.png)

![image](https://user-images.githubusercontent.com/39102955/51323537-8bc7c180-1a79-11e9-9e25-daed4a6dc883.png)

![image](https://user-images.githubusercontent.com/39102955/51323613-bca7f680-1a79-11e9-8ba4-c37a875833f8.png)


@wasswa-derick @solnsubuga @3Nakajugo @StephenCMwanika @bashman1 @NicholusMuwonge @66hakeem @Phionanamugga @engjames @Paphra @digit2018 @BrunoNicholas @edrinesolo @astonetuhame @jngisiro @Hassan-Ssemanda @Emmanuelaaron @alexxsanya @dkato8181 @AbuDeCoder @Nico-Nsereko @alinitweSandra @daspro1503 @ivankfit @ asimwec @brucenelm @sebbss @Rhytah @llwasampijja @musasizifrancis @RachealN @nearjay06 @mderek227 @sanya-kenneth @neelxie @frankopkusianwar @AtamaZack @ManuelDominic @KabohaJeanMark @mwinel @chrismatgit @ringtho @nadralia @Kwandera @JammyKalungi @reifred @jennizalwango @jkMugabe @richien @Araali1 @Mastula77 @Genza999 @wabuluka @noahkalyesubula @lizz24 @kalsmic @k7ko @bekeplar @e-ian @masete @Manorlds-Eaglespark @nanfuka @HarrietAkot @PatrickMugayaJoel @kyakusahmed @Anguandia @CandySusan @kabaliisa @bisonlou @HabibSentongo @v1b3m @kdenno @davicken @dtnnml @Araali1 @natalie-abbie @ernesthenry
